libs/wallet-admin/template/mocks.ts

```diff
@@ -7,7 +7,7 @@ export function MockAPIRequest (id: Identifier.<%= getMethodName(method) %>, par
 export function MockAPIRequest (id: Identifier) {
```

Comment:
Line 2:

```typescript
import { WalletModel, Identifier } from './'
```

would be more correct as:
```typescript
import { Identifier } from './'
import type { WalletModel } from './
```

Reply:
in this case, it is ultimately the same. i think this only matters when you're importing an enum but using it only as a type and not a value. here i'm importing a type and an enum, the type is going to be stripped out on transpile, the enum is used as a value in the switch cases.

Reply:
It might be true, but it will be highlighted by https://typescript-eslint.io/rules/consistent-type-imports/ . I don't mind either way :)